apps/database/migrations/20220803095315_add_plans.js

```diff
@@ -0,0 +1,35 @@
+exports.up = async (knex) => {
+  await knex.schema.createTable('plans', (table) => {
+    table.bigincrements('id').primary()
+    table.dateTime('createdAt').notNullable()
+    table.dateTime('updatedAt').notNullable()
```

Comment:
I think there is a timestamps option to create updatedAt/createdAt at the same time but fine.

Reply:
`table.timestamps(true, true)` but never used in the project until now